.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,117 @@
+name: Bug Report
+description: Create a report to help us improve
+title: '[BUG]: '
+labels: ['Bug']
+body:
+  - type: checkboxes
+    id: check-duplicates
+    attributes:
+      label: Is this a duplicate?
+      description: Check for duplicate issues.
+      options:
+        - label: I confirmed there appear to be no [duplicate issues](https://github.com/NVIDIA/cccl/issues) for this bug and that I agree to the [Code of Conduct](CODE_OF_CONDUCT.md)
+          required: true
+          
+  - type: dropdown
+    id: bug-type
+    attributes:
+      label: Type of Bug
+      description: What kind of bug are you running into?
+      multiple: false
+      options:
+        - Silent Failure
+        - Runtime Error
+        - Compile-time Error
+        - Performance
+        - Something else
+    validations:
+      required: true
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what problem you are running into.
+      placeholder: "cuco::static_map::insert fails to insert all keys."  
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: How to Reproduce
+      description: Steps used to reproduce the bug.
+      placeholder: |
+        1. Construct a cuco::static_map<int,int> with capacity for 100,000 elements.
+        2. Insert 50,000 unique key-values.
+        3. Perform '`find`' for all the same keys.
+        4. Not all keys are found.
```

```suggestion
        0. See godbolt link below for exact reproducer
        1. Construct a `cuda::std::tuple`
        2. Use structured bindings to destructure the elements of the tuple.
        3. Compilation fails with the error message:
        ```
<source>(5): error: cannot bind to non-public member "cuda::std::__4::tuple<_Tp...>::__base_ [with _Tp=<int, int>]"
    auto [a,b] = t;
```

```